conformance/utils/suite/suite.go

```diff
@@ -53,9 +53,6 @@ const (
 	// This option indicates support for Destination Port matching (extended conformance).
 	SupportRouteDestinationPortMatching SupportedFeature = "RouteDestinationPortMatching"
 
-	// This option indicates GatewayClass will update the observedGeneration in it's conditions when reconciling
-	SupportGatewayClassObservedGenerationBump SupportedFeature = "GatewayClassObservedGenerationBump"
```

Comment:
Maybe mark this `Deprecated` for a single release and then drop it in a subsequent one